pkg/etcdutil/etcd.go

```diff
@@ -52,8 +52,15 @@ func NewClientConfig(s *state.State, host kubeoneapi.HostConfig) (*clientv3.Conf
 		return nil, fail.Etcd(err, "TLS config creating")
 	}
 
+	var endpoints []string
+	if s.Cluster.ClusterNetwork.IPFamily.IsIPv6Primary() {
+		endpoints = []string{fmt.Sprintf("[%s]:2379", host.IPv6Addresses[0])}
```

Comment:
Maybe we should verify that this IPv6 address/element of the slice exists to avoid eventual NPEs. I know we have some validation for this, but better to be on the safe side.